src/core/sal/bochs/BochsController.cc

```diff
@@ -27,7 +27,7 @@ BochsController::BochsController()
 
 BochsController::~BochsController()
 {
-	delete m_Mem;
+	delete &getMemoryManager();
```

Comment:
At this point only the MemoryManager for MEMTYPE_RAM would be deleted, since this is the default for getMemoryManager().